e2e/Makefile

```diff
@@ -26,8 +26,10 @@ check-token:
 	  exit 1; \
 	fi
 
-install-terraform:
-	sudo apt-get install wget unzip
-	wget https://releases.hashicorp.com/terraform/0.11.13/terraform_0.11.13_linux_amd64.zip
-	unzip terraform_0.11.13_linux_amd64.zip
-	sudo mv terraform /usr/local/bin/
+install-dependencies:
```

Comment:
i really liked how we got the CAPL makefile looking with the individual targets, goarch, version pins and go installs. since we're changing this any way can we move to that style?